rust_src/src/buffers.rs

```diff
@@ -382,7 +382,7 @@ pub fn set_buffer(buffer_or_name: LispObject) -> LispObject {
 /// `inhibit-read-only' text property set, the error will not be raised.
 #[lisp_fn(min = "0")]
 pub fn barf_if_buffer_read_only(position: Option<EmacsInt>) -> () {
-    let pos = position.unwrap_or_else(point());
+    let pos = position.unwrap_or_else(|| point());
```

Comment:
minor details :)

Reply:
Technically I could have passed just 'point' but I think the lambda reads better.